module-about/src/main/java/com/orange/ods/module/about/ui/navigation/OdsAboutNavigation.kt

```diff
@@ -37,9 +37,9 @@ import com.orange.ods.module.about.ui.configuration.OdsAboutUrlMenuItem
 import com.orange.ods.module.about.ui.utilities.extension.launchUrl
 
 
-private object OdsAboutDestinations {
-    const val HomeRoute = "ods/module/about/home"
+object OdsAboutDestinations {
     const val AboutRoute = "ods/module/about/"
```

Comment:
Should we rename it `AboutNavRoute` or something like this cause this is the route of the nested graph?